exercise5-2.go

```diff
@@ -8,19 +8,24 @@ import (
 	"math"
 	"math/rand"
 	"os"
+	"time"
 )
 
-var palette = []color.Color{color.White, color.Black}
+func main() {
+	lissajous2(os.Stdout)
+}
+
+var palette2 = []color.Color{color.Black, color.RGBA{255, 30, 64, 1}, color.RGBA{0, 230, 64, 2}, color.RGBA{20, 50, 230, 3}}
```

Comment:
I'm not sure it makes sense to name this `palette2` when the original `palette` variable no longer exits.  It isn't functional wrong, just confusing.  Ditto for the `*Index2` and `out2` variables below.

Does it matter in the context of learning code that others aren't likely to look at?  Not that much.  But one reason I'm reviewing these is to teach you about our expectations for production code and this definitely would get called out there.

Reply:
Yeah, I realize it doesn't make sense in this case, however in the context of VS code, it kept giving my errors saying the variable was already used in another file. So these variable are more so for myself so I didn't have to see the red lines.

Reply:
This is indicative of a deeper problem with your current code structure across exercises.

Why is VSCode complaining about variables being reused between these two files?
Is it a desirable thing that all variables outside a function can't be duplicated across these files?
What will when a problem is big enough to be split into two files (e.g. a main and a library or test file)?
What is a better layout/file structure for these problems?

Reply:
Also, to be fair, this isn't covered in depth until TGPL § 2.6.  If you'd like to wait until then we can postpone this discussion.